app/src/main/java/org/p2p/wallet/settings/ui/network/SettingsNetworkFragment.kt

```diff
@@ -26,44 +26,46 @@ class SettingsNetworkFragment :
     SettingsNetworkContract.View {
 
     companion object {
-        fun create(requestKey: String, resultKey: String): SettingsNetworkFragment = SettingsNetworkFragment().withArgs(
-            EXTRA_REQUEST_KEY to requestKey,
-            EXTRA_RESULT_KEY to resultKey
-        )
+        fun create(requestKey: String, resultKey: String): SettingsNetworkFragment =
+            SettingsNetworkFragment()
+                .withArgs(
+                    EXTRA_REQUEST_KEY to requestKey,
```

Comment:
Can me explain somebody, we use EXTRA or we use ARG, cause Roma use ARG in latest PR, you use EXTRA, which one is actual ?

Reply:
oh it's just legacy from the old class, i will rename